src/whenever/__init__.py

```diff
@@ -1290,15 +1290,15 @@ def __init__(
             minute,
             second,
             microsecond,
-            ZoneInfo(tz),
+            zone_info := ZoneInfo(tz),
```

Comment:
Did you suspect I would prefer this? (I do.) Or do you like using the walrus here as well 👀 ?